src/lwc/healthCheckDisplay/healthCheckDisplay.js

```diff
@@ -0,0 +1,22 @@
+import { LightningElement, api, track } from 'lwc';
+
+import stgLoadingSpinnerText from '@salesforce/label/c.stgLoadingSpinnerText';
+
+export default class HealthCheckDisplay extends LightningElement {
+    @api healthCheckDefinitionsList;
+    
+    @track loadedHealthCheckDefCount = 0;
+    @track isDisplayHealthCheckGroup = false; // determines whether healthcheckgroup card is displayed
```

Comment:
If this variable is necessitating a comment, is there a name for it that would remove that necessity?